### PR TITLE
Clarify admin & FPF roles and responsibilities; services

### LINKED ISSUE #200: Clarify FPF/news org responsibilities in security incidents
Our documentation currently only speaks in a limited manner to incident response responsibilities and procedures. The [minimum security requirements](https://docs.securedrop.org/en/stable/deployment/minimum_security_requirements.html) specify:
> An established monitoring plan and incident response plan. Who will receive the OSSEC alerts and what will their response plan be? These should cover technical outages and a compromised environment plan.

In addition, the [admin responsibilities](https://docs.securedrop.org/en/stable/admin.html#responsibilities) section mentions monitoring of OSSEC alerts and applying updates.

We should define more clearly the division of responsibilities between the SecureDrop development team and SecureDrop users when it comes to the concrete response to a security incident. That may include:
- disclosure and notification procedures and channels
- accepted processes for distributing patches or point releases
- obligations of admins in cases where manual action is required.

### LINKED ISSUE #203: Mention support portal in "Getting Support" chapter
https://docs.securedrop.org/en/stable/getting_support.html currently emphasizes Gitter and the forums. In practice, we tend to direct people to the support portal, and do not charge for membership (we do charge for priority support). We should make this more explicit in the docs.

### PR DESCRIPTION
**Pull request by eloquence**
## Status
Ready for review

## Description of Changes

Resolves #200 
Resolves #203

## Checklist (Optional)

- [x] Doc linting (`make docs-lint`) passed locally
- [ ] Doc link linting (`make docs-linkcheck`) passed
- [x] You have previewed (`make docs`) docs at http://localhost:8000

### CODE REVIEW

File: docs/admin.rst
Comment (emkll) [inline docs/admin.rst:21]: and firewall

File: docs/admin.rst
Comment (emkll) [inline docs/admin.rst:47]: I would lead this with something like

> The org hosting SecureDrop owns and operates their own instances

File: docs/admin.rst
Comment (emkll) [inline docs/admin.rst:61]: For security incidents we will likely issue a release, I would say something

> On extremely rare occasion when a fix is extremely time sensitive, we may provide signed patches to impacted SecureDrop instances to allow for immediate resolution

File: docs/admin.rst
Comment (emkll) [inline docs/admin.rst:63]: Also a line stating if there's a bug with the system, to reach out to us to inform us (since others can have the same issue) and to avoid using guidance that is not SecureDrop-specific or not from a source that is vetted by us as it can interfere or may introduce security risks.

File: docs/admin.rst
Comment (emkll) [inline docs/admin.rst:22]: and firmware(s)

File: docs/admin.rst
Comment (emkll) [inline docs/admin.rst:23]: and physical security

**Review by emkll — COMMENTED** [review]
Some comments inline. Once they are resolved, good to merge from my perspective, after review from @rocodes or @zenmonkeykstop

File: docs/admin.rst
Comment (eloquence) [reply docs/admin.rst:22]: created a separate bullet for this in 3c6dd8f5e6106c4044ce90451c36efe96e18b56a, we may want to flesh out instructions for monitoring/applying firmware updates

File: docs/admin.rst
Comment (eloquence) [reply docs/admin.rst:23]: added in 3c6dd8f5e6106c4044ce90451c36efe96e18b56a

File: docs/admin.rst
Comment (eloquence) [reply docs/admin.rst]: Added language that covers reporting bugs and avoiding unvetted docs when in doubt (didn't want to discourage peer support via forums etc., so trying to strike a balance) in 69eac636a4740e600b88853c7838764b9c60039d

File: docs/admin.rst
Comment (eloquence) [reply docs/admin.rst]: Reworded consistent with the above in 69eac636a4740e600b88853c7838764b9c60039d

File: docs/admin.rst
Comment (eloquence) [reply docs/admin.rst]: Used similar language in 69eac636a4740e600b88853c7838764b9c60039d

File: docs/admin.rst
Comment (eloquence) [reply docs/admin.rst]: added via 69eac636a4740e600b88853c7838764b9c60039d

**Review by rocodes — APPROVED** [review]
LGTM!

**Pull request merged**
